src/passes/shadowstack.ts

```diff
@@ -162,6 +162,18 @@ function matchTostack(module: Module, expr: ExpressionRef): ExpressionRef {
   return 0;
 }
 
+/** Tests whether a `value` matched by `matchTostack` needs a slot. */
```

Comment:
nit: couldn't `matchToStack` will be more readable than `matchTostack`?

Reply:
This one is a little strange, yeah, because the pattern it looks for is `__tostack`, all lower case, so capitalizing that makes `matchToStack` :)

Reply:
I don't think internal stubbed name (`__tostack`) and actual ABI (`matchTostack`) should strictly follow same convention)

Reply:
Otherwise it should be something like `match__tostack` =)

Reply:
Perhaps just `matchPattern`, to match the pattern we look for, independent of its name? :)

Reply:
Yeah, also ok